.github/workflows/deploy.yml

```diff
@@ -0,0 +1,58 @@
+# Deploy workflow - triggered by workflow_run after successful build
+# This workflow has access to secrets but never executes untrusted code
+# It only downloads and deploys pre-built artifacts from the build workflow
+# Security: Fork code cannot access secrets as it only runs in build workflow
+# Deploys to IPFS for all branches
+
+name: Deploy
+
+# Explicitly declare permissions
+permissions:
+  contents: read
+  pull-requests: write
+  statuses: write
+
+on:
+  workflow_run:
+    workflows: ["Build"]
+    types: [completed]
+
+env:
+  BUILD_PATH: 'docs-build'
```

Comment:
Where is the `docs-builds` constant coming from?

Reply:
just arbitrary name -- in `deploy.yml` we download  and unpack `build.yml` result to  `BUILD_PATH` and then `ipfs-deploy-action` reads from it